tests/api/groups/update_settings.test.js

```diff
@@ -97,4 +90,108 @@ describe('groups:update-settings', () => {
     const { group } = await publicReq('get', `/api/groups?action=by-id&id=${groupId}`)
     group.open.should.be.true()
   })
+
+  describe('notifications', () => {
+    it('should send a notification to groups members when updating the name', async () => {
+      const { group, member } = await createGroupWithAMember()
+      const { _id: groupId, name } = group
+      const updatedName = `${name}-updated`
+      await authReq('put', endpoint, {
+        group: groupId,
+        attribute: 'name',
+        value: updatedName
+      })
+      const notifications = await getNotifications({ user: member, type: 'groupUpdate', subject: groupId })
+      const notification = notifications.find(isAttributeNotification('name'))
+      notification.data.attribute.should.equal('name')
+      notification.data.previousValue.should.equal(name)
+      notification.data.newValue.should.equal(updatedName)
+    })
+
+    it('should send a notification to groups members when updating the description', async () => {
+      const { group, member } = await createGroupWithAMember()
+      const { _id: groupId, description } = group
+      const updatedDescription = `${description}-updated`
+      await authReq('put', endpoint, {
+        group: groupId,
+        attribute: 'description',
+        value: updatedDescription
+      })
+      const notifications = await getNotifications({ user: member, type: 'groupUpdate', subject: groupId })
+      const notification = notifications.find(isAttributeNotification('description'))
+      notification.data.attribute.should.equal('description')
+      notification.data.previousValue.should.equal(description)
+      notification.data.newValue.should.equal(updatedDescription)
+    })
+
+    it('should send a notification to groups members when updating the searchable flag', async () => {
+      const { group, member } = await createGroupWithAMember({ searchable: false })
+      const { _id: groupId } = group
+      await authReq('put', endpoint, {
+        group: groupId,
+        attribute: 'searchable',
+        value: true
+      })
+      const notifications = await getNotifications({ user: member, type: 'groupUpdate', subject: groupId })
+      const notification = notifications.find(isAttributeNotification('searchable'))
+      notification.data.attribute.should.equal('searchable')
+      notification.data.previousValue.should.equal(false)
+      notification.data.newValue.should.equal(true)
+    })
+
+    it('should send a notification to groups members when updating the open flag', async () => {
```

Comment:
i dont think testing every single attributes is necessary (even though it was nice for development purposes), now its feels its only redundant tests that takes space. Maybe it could be unit tests instead
On this file maybe it would be nice to test what happen if attribute's value if of a wrong type, ie. what kind of error shall it throw